modules/cnv-connecting-resource-bridge-network.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv-attaching-vm-multiple-networks.adoc
+
+[id="connecting-resource-bridge-network-{context}"]
+= Connecting a resource to a bridge-based network
+
+As a network administrator, you can configure a NetworkAttachmentDefinition
+of type `cnv-bridge` to provide Layer-2 networking to Pods and virtual machines.
+
+.Prerequisites
+
+* {ProductName} 2.0 or newer
+* A Linux bridge must be configured and attached to the correct
+Network Interface Card on every node.
+* If you use VLANs, `vlan_filtering` must be enabled on the bridge.
+* The NIC must be tagged to all relevant VLANs.
+** For example: `bridge vlan add dev bond0 vid 1-4095 master`
+
+.Procedure
+
+. Create a new file for the NetworkAttachmentDefinition in any local directory.
+The file must have the following contents, modified to match your
+configuration:
++
+[source,yaml]
+----
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: a-bridge-network
+spec:
+  config: '{
+    "cniVersion": "0.3.0",
+    "name": "a-bridge-network", <1>
+    "type": "cnv-bridge", <2>
+    "bridge": "br0", <3>
+    "isGateway": true,
+    "ipam": {}
+}'
+----
+<1> The `name` value is part of the annotation you will use in the next step.
```

Comment:
There is inconsistency between the callout pointer (hope I'm using a correct and comprehensible terminology) and the reference callout itself. For example:
This is how the callout pointer looks:
`"name": "a-bridge-network", (1)`
and this is how the referenced callout looks:
`i. The name value is part of the annotation you will use in the next step.`
I would expect the index in the callout to be also "(1)", and not "i".
I'm not an expert in tech-writing and documentation, but I think it doesn't look good.

Reply:
Ah, are you rendering this in Github? It shouldn't look like that. 

I just uploaded a local build which is a good approximation of what it will look like when published: http://file.bos.redhat.com/pousley/042919/cnv-127/cnv/cnv-attaching-vm-multiple-networks.html

Reply:
OK, it does look great on your upload.
You can resolve this conversation (I can't find how to resolve conversations myself).